generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -208,7 +211,6 @@ sonar-analyze:
         - frontend-test
         <%_ } _%>
     script:
-        # - ./mvnw compile -Dmaven.repo.local=$MAVEN_USER_HOME
```

Comment:
@atomfrede do you know what is the motivation of the compile phase before sonar since the classes are part of artifacts in compile stage?

Reply:
Good point. I have no explanation. First I thought to be able to execute sonar out of order but everything should be in the cache/artifacts of the build